src/data/use-cases/db-forget-password.ts

```diff
@@ -1,17 +1,24 @@
-import { type LoadGuardianByEmailRepository } from '@/data/protocols/guardian'
-import { type TokenGenerator } from '../protocols'
-import { type EmailService } from '@/domain/use-cases'
-import { type ForgetPassword } from '@/domain/use-cases/forget-password'
+import {
+  type LoadGuardianByEmailRepository,
+  type HashGenerator,
+  type UpdateVerificationTokenRepository,
+  type TokenGenerator
+} from '@/data/protocols'
+import { type EmailService, type ForgetPassword } from '@/domain/use-cases'
 
 export class DbForgetPassword implements ForgetPassword {
   private readonly loadGuardianByEmailRepository: LoadGuardianByEmailRepository
+  private readonly updateVerificationTokenRepository: UpdateVerificationTokenRepository
```

Comment:
will this be a repository after the main update?

Reply:
Yes, I will refactor on merge. Because the branch refactor/repository-pattern still not aproved, so I followed the old pattern.